crates/sargon/src/profile/logic/persona/query_personas.rs

```diff
@@ -0,0 +1,35 @@
+use crate::prelude::*;
+
+impl Profile {
+    /// Returns the non-hidden personas on the current network, empty if no personas
+    /// on the network
+    pub fn personas_on_current_network(&self) -> Result<Personas> {
+        self.current_network().map(|n| n.personas.non_hidden())
```

Comment:
@sergiupuhalschi-rdx sync with @micbakos-rdx about this :D who ever merges first... https://github.com/radixdlt/sargon/blob/0e4ba5a48cc0e2923ae94f4ae57ebf5c2e5c1fa7/crates/sargon/src/profile/logic/account/query_personas.rs#L6